Source/WebKit/UIProcess/API/glib/WebKitWebsiteDataManager.cpp

```diff
@@ -486,6 +492,25 @@ static void webkit_website_data_manager_class_init(WebKitWebsiteDataManagerClass
             nullptr, nullptr,
             FALSE,
             static_cast<GParamFlags>(WEBKIT_PARAM_READWRITE | G_PARAM_CONSTRUCT_ONLY)));
+
+
+    /**
+     * WebKitWebsiteDataManager:per-origin-storage-quota:
+     *
+     * Max size of origin data storage. This includes CacheStorage and IndexedDB.
+     * Value of 0 means no change so whatever the default value is it will be used.
+     *
+     * Since: 2.40
+     */
+    g_object_class_install_property(
+        gObjectClass,
+        PROP_PER_ORIGIN_STORAGE_QUOTA,
+        g_param_spec_uint64("per-origin-storage-quota",
+            _("Per origin storage quota"),
+            _("The maximum size of storage per origin (CacheStorage and IndexedDB)"),
```

Comment:
Ditto regarding not mentioning the particular data types managed by this property